toolkit/tools/imagecustomizerapi/previewfeaturetype.go

```diff
@@ -22,11 +22,15 @@ const (
 
 	// PreviewFeatureSnapshotTime enables support for snapshot-based package filtering.
 	PreviewFeaturePackageSnapshotTime PreviewFeature = "package-snapshot-time"
+
+	// PreviewFeatureCrashDump enables support for crash dump configuration.
+	PreviewFeatureCrashDump PreviewFeature = "crash-dump"
```

Comment:
The field is called `kdumpBootFiles` but the preview flag is called `crash-dump`. Perhaps the preview flag should be called `kdump`?